src/config/config.php

```diff
@@ -14,6 +14,9 @@
  * permissions and limitations under the License.
  */
 
+use Illuminate\Foundation\Application;
```

Comment:
This is only a dev dependency, so it shouldn't be used anywhere in the `src/` directory. (I just removed it from the 3.0 branch for this reason.)